File: Problem1.java

class Problem1 {
    public static void main(String[] argv) {
        int limit = 1000;
        int sum = 0;

        for(int i = 0; i < limit; i++) {
            if(i%5==0 || i%3==0) {
                sum += i; 
            }
        }

        System.out.println(sum);
    }
}

File: Problem1.rs

fn main() {
    
    let limit = 1000;
    let mut sum = 0;

    for x in 1..limit {
        if x % 3 == 0 || x % 5 == 0 {
            sum = sum + x; 
        } 
    }

    println!("Sum is: {}", sum);
}
